aten/src/ATen/native/native_functions.yaml

```diff
@@ -1091,6 +1091,9 @@
   dispatch:
     CompositeExplicitAutograd: count_nonzero
 
+- func: cov(Tensor self, Tensor? other=None, bool? rowvar=True, bool? bias=None, int? ddof=None, Tensor? fweights=None, Tensor? aweights=None) -> Tensor
```

Comment:
We could make everything keyword only after other. Also, ddof can replace bias entirely and thus we can probably do without bias and instead educate people in the docs that if they want bias to be False they can set ddof to 1.